.github/workflows/release.yml

```diff
@@ -42,24 +45,30 @@ jobs:
         with:
           version: ${{ steps.previous_tag.outputs.tag }}
 
-      - name: Set development manifest
-        if: steps.semvers.outputs.v_patch == steps.get_current_tag.outputs.tag
-        run: |
-            jq --argjson icons "{\"16\": \"icons/dev/16x16.png\",\"48\": \"icons/dev/48x48.png\",\"128\": \"icons/dev/128x128.png\"}" '.icons = $icons | .browser_action.default_icon = $icons | .name = "Liquality Wallet - Dev"' ./src/manifest.json > ./src/manifest.tmp
-            mv ./src/manifest.tmp ./src/manifest.json
-
-      - name: Set development agents
-        if: steps.semvers.outputs.v_patch == steps.get_current_tag.outputs.tag
+    # Set the Amplitude Analytics only for Major prod releases.
+      - name: Set development agents (major for Prod)
+        if: steps.semvers.outputs.v_major == steps.get_current_tag.outputs.tag
```

Comment:
Shouldn't it be `v_minor` ?
`major.minor.patch`
we release to prod on every `minor`